public/js/script.js

```diff
@@ -41,4 +70,5 @@ function loadEnd(event) {
   console.log(fastQCFile);
   linePlot.render(fastQCFile.modules.qual.quintiles);
   boxPlot.render(fastQCFile.modules.qual.quintiles);
+  parseBasicInfo(fastQCFile.modules.basic)
 }
```

Comment:
See the way I've done the tables in the multiple-files zip I shared on Gitter -- that's a cleaner more 'jQuery' way of doing tables. You'll also need to add something like ''$('#basic-table-insert').empty()'' to the start of the function so the table doesn't get added to the DOM multiple times if the function gets called more than once. :)